### ISSUE TITLE
Is this alive?

### ISSUE THREAD

**Issue by xaralis** [socialradar/batch-request#16]
Is this project still being maintained? There are some security-related issues to be fixed. 

```
Name       Installed   Patched  Vulnerable Dependency
validator    2.0.0    >=3.22.1  xxx@xxx >  batch-request@0.1.1 > validator@2.0.0
qs           0.6.6     >= 1.x   xxx@xxx >  batch-request@0.1.1 > request@2.27.0 > qs@0.6.6
```

Theoretically, this should be very simple to do, just update dependencies. I'm willing to make a PR in case someone will merge it (not sure about it, since there are other PR waiting in queue).

**Comment by tsheaff** [comment]
@xaralis I don't think SocialRadar is still maintaining this, no.  It's a great library tho, and we're using it in production.  I've forked it just now to fix #18 and my fork lives here: https://github.com/calmcom/batch-request

Not sure how it would work to get permissions to update the version on npm, but my fork has bumped up the version to 0.1.2

@victorquinn 
@asilvas 
@ghafran 
@petkaantonov 

Are any of you able to give me permissions to push the npm package?  We're happy to maintain this at [Calm](https://www.calm.com/) going forward.

**Comment by victorquinn** [comment]
Hey all, we are still maintaining it, apologies I missed the security updates here. @tsheaff if you just submit a PR I'll merge it ASAP.

Apologies it seemed neglected, I just noticed now that I wasn't signed up for notifications on this repo for some reason. Your mentioning me hit my email though and I now see that there have been PRs and issues, I'll do my best to clean things up over the next couple days!

**Comment by victorquinn** [comment]
FYI I merged the PR which fixes #18 and pushed 0.1.2 live to npm! Apologies again on looking unresponsive here.

Will fix the tests and update these outdated dependencies.

**Comment by tsheaff** [comment]
Awesome, thanks a bunch @victorquinn and nice job on the concept and documentation. It's been super helpful for us at Calm.
